helpers/deploy/deploymentConfig.ts

```diff
@@ -178,6 +178,12 @@ export const config: Config = {
       { method: "setMaxDailyReceiveLimit(uint256)", args: [100] },
     ],
   },
```

Comment:
I think we can add  [here](https://github.com/VenusProtocol/governance-contracts/blob/develop/helpers/deploy/deploymentConfig.ts#L103) new network configuration as well, for record keeping purpose.

Reply:
https://github.com/VenusProtocol/governance-contracts/pull/104/commits/292a9096cad94f495a5d7d29fd8db4cf491a1805